docs/source/tutorial/tutorial_2.mdx

```diff
@@ -549,32 +549,38 @@ case .loading:
+<img src="images/tap_to_load_more.png" alt="Screenshot with loading cell" width="300">
+</img>
+
+
+When you tap that cell, the rows will load and then redisplay. If you tap it several times, it reaches a point where the loading cell is no longer displayed, and the last launch was SpaceX's original FalconSat launch from Kwajalien Atoll:
+
+
+<img src="images/list_all_loaded.png" alt="List with all launches loaded and no loading cell" width="300">
+</img>
 
-![list with all launches loaded and no loading cell](images/list_all_loaded.png)
 
-Congratulations, you've now loaded all of the possible launches! But when you tap one, you'll still get the same boring detail page. 
+Congratulations, you've loaded all of the possible launches! But when you tap one, you still get the same boring detail page. 
 
-Next, you'll make the detail page a lot more interesting by taking the ID returned by one query and passing it to another!
+Next, you'll make the detail page a lot more interesting by taking the ID returned by one query and passing it to another.
 
-## Getting more details for the detail page
+## Add more details to the detail view
 
-To get more information to show on the detail page, you could do a couple of different things: 
+To get more information to show on the detail page, you have a couple of options: 
 
-- You could request all the details you want to display for every single  of the launches in the `LaunchList` query, and then pass that retrieved object on to the `DetailViewController`. 
-- You could use the identifier of the launch to request all the details you want to display just for the selected launch using a different query. 
+- You could request all the details you want to display for every single launch in the `LaunchList` query, and then pass that retrieved object on to the `DetailViewController`. 
+- You could provide the identifier of an individual launch to a _different_ query to request all the details you want to display. 
 
-The first option *can* seem easier if there's not a substantial difference in size between what you're requesting for a list vs. a detail page. 
+The first option *can* seem easier if there isn't a substantial difference in size between what you're requesting for the list versus the detail page. 
 
-However, remember that part of the advantage of GraphQL over REST is that you only need to get the minimum amount of data needed to display a certain page. If you're not going to be displaying additional information, how much of your user's battery and bandwidth can you save, and how many unnecessary database calls on your backend can you prevent by not asking for something until you need it?
+However, remember that one of the advantages of GraphQL is that you can query for _exactly_ the data you need to display on a page. If you're not going to be displaying additional information, you can save bandwidth and execution time on both your frontend and backend by not asking for data until you need it.
```

Comment:
Would definitely like to keep the battery thing in here - less bandwidth = less battery and that's critical for mobile